python/GafferUI/ContextVariablesTweaksUI.py

```diff
@@ -0,0 +1,182 @@
+##########################################################################
+#
+#  Copyright (c) 2022, Cinesite VFX Ltd. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of John Haddon nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import imath
+import six
+import functools
+
+import IECore
+
+import Gaffer
+import GafferUI
+
+Gaffer.Metadata.registerNode(
+
+	Gaffer.ContextVariablesTweaks,
+
+	"description",
+	"""
+	Makes modifications to context variables.
+	""",
+
+	plugs = {
+
+		"ignoreMissing" : [
+
+			"description",
+			"""
+			Ignores tweaks targeting missing context variables. When off, missing context
+			variables cause the node to error, unless the tweak mode is `CreateDefault`.
+			""",
+			"nodule:type", "",
+
+		],
+
+		"tweaks" : [
+
+			"description",
+			"""
+			The tweaks to be made to the context variables. Arbitrary numbers of user defined
+			tweaks may be added as children of this plug via the user interface, or
+			using the ContextVariablesTweaks API via python.
+			""",
+
+			"plugValueWidget:type", "GafferUI.LayoutPlugValueWidget",
+			"layout:customWidget::footer:widgetType", "GafferUI.ContextVariablesTweaksUI._TweaksFooter",
+			"layout:customWidget:footer:index", -1,
+
+			"nodule:type", "",
+
+		],
+
+		"tweaks.*" : [
+
+			"tweakPlugValueWidget:allowCreateDefault", True,
```

Comment:
I think we want to allow `Create` mode too (see related comments in TweakPlugValueWidget).

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5047/commits/19418b4fa6215cc4afabe83fbd08f1c8acb24563